src/local-introspection.js

```diff
@@ -3,39 +3,39 @@ const JwksClient = require('jwks-rsa');
 const { jwk2pem } = require('pem-jwk');
 const jwt = require('jsonwebtoken');
 const { promisify } = require('util');
+const assert = require('assert');
 const errors = require('./errors');
 
 const jwtVerify = promisify(jwt.verify);
 
 module.exports = (options) => {
-  let jwksFetchKey = null;
+  let jwksClient = null;
   if (options.jwks && options.jwks.keys) {
     debug('Configured JWKS with static keys');
     const keys = {};
     options.jwks.keys.forEach((k) => {
       keys[k.kid] = jwk2pem(k);
     });
-    jwksFetchKey = async function jwksFetchStaticKey(keyId) {
-      if (keys[keyId]) {
-        return { key: keyId, nbf: null, rsaPublicKey: keys[keyId] };
-      }
-      throw new errors.IntrospectionError('Unable to find key');
+    jwksClient = {
+      getSigningKey: async function jwksFetchStaticKey(keyId) {
+        assert.ok(keys[keyId], new errors.IntrospectionError('Unable to find key'));
+        return { key: keyId, rsaPublicKey: keys[keyId], getPublicKey: () => keys[keyId] };
```

Comment:
Looks like only `getPublicKey`  is used, can `rsaPublicKey` be removed?

Reply:
Yeah, probably. Left it as it is in the key object that the jwks client returns too.